packages/account/src/BaseSmartAccount.ts

```diff
@@ -47,9 +47,18 @@ export abstract class BaseSmartAccount implements IBaseSmartAccount {
     this.overheads = _smartAccountConfig.overheads;
     this.entryPointAddress = _smartAccountConfig.entryPointAddress ?? DEFAULT_ENTRYPOINT_ADDRESS;
     this.accountAddress = _smartAccountConfig.accountAddress;
-    this.bundler = _smartAccountConfig.bundler;
+
     this.chainId = _smartAccountConfig.chainId;
 
+    if (_smartAccountConfig.bundlerUrl) {
```

Comment:
we can also do conditional props for these? or an overkill
@AmanRaj1608 @VGabriel45 @joepegler